### DISCUSSION TITLE
Toast to the Pizza CLI Launch - 9/16/24

### DISCUSSION THREAD

**Discussion by jpmcb** [open-sauced/feedback#42] [Show and tell]
Today, we're celebrating the launch of our new CLI tool – a solution designed to simplify code ownership, enhance collaboration across engineering teams, and integrate deeply with the OpenSauced platform.

This CLI addresses a common challenge in scaled open source and inner source projects of identifying code owners and streamlines the process of managing large, complex codebases. From automatically generating `CODEOWNERS` files to providing clear insights into your repository structure, we're very excited to be bringing you this powerful tool!

Read about the Pizza CLI:

- [docs](https://opensauced.pizza/docs/tools/pizza-cli/)
- [Introducing the Pizza CLI](https://opensauced.pizza/blog/introducing-the-pizza-cli)

# How to toast

Take a moment to celebrate with us!

Share a photo of yourself with your favorite pizza of pizza or drink. Post it as a comment below, and give a shout-out to someone on the team who contributed to this project or share how you think it will improve your development workflow!!!

**Comment by jpmcb** [comment]
![Screenshot 2024-09-16 at 2 56 31 PM](https://github.com/user-attachments/assets/f6e61af6-5fee-435b-8e66-5fc2fe0909e9)

Congratulations to the whole OpenSauced team for turning this idea into a reality. We took some massive feedback from industry leaders and turned around a working CLI in a matter of a few weeks.

- MASSIVE ups to the entire @open-sauced/engineering team for picking up Go and Cobra really quickly. I'm impressed with all the Go code we cut in a short amount of time and how many features we were able to get done.
- @bdougie for leading us through the forest to find a path forward on some really big ideas. Without your guidance and leadership, we wouldn't have these tools today.
- @brandonroberts - kudos for quickly ramping up on Go, getting some bugs squashed, but also juggling ALOT of extra stuff as my focus was targeted only on this CLI. It can be thankless work keeping things going and the ship afloat, but you've put in alot of efforts that have kept things in tip top shape!!! (cough ingress cough 😉)
- @zeucapua - I'm very very impressed with the Go code you cut in and your ability to quickly learn new technologies. You went from Gopher novice to getting some really big beefy features in. This is definitely something to be very proud of! All that and you kept getting some big design features in on the App! Huge!!
- @nickytonline - This launch wouldn't have happened without you coming alongside me and going deep on Go, the Pizza-action, our release workflows, docs, cutting in features, fixing bugs, and all around helping me get things to a point where we could release binaries to an audience! Massive effort!
- @BekahHW - I really appreciate all the product, positioning, and content feedback you've given me. You've become a trusted compatriot to me and have helped me gain more and more confidence in my writing! All the docs you reviewed, content you helped get on the content calendar, and everything else was a incredible help in this launch.
- @isabensusan - You've been an incredible sounding board on all the ideas we end up having: it can be so easy to get caught up in the process of coding an MVP out and forget the user experience. You've been incredibly fundamental in grounding our efforts in this launch!

**Comment by bdougie** [comment]
Thanks to all the @open-sauced/staff for coming together in the matter of a few weeks to test and validate this feature. Not only was this a new experience being command line first, but the team really came together with the Go code. I look forward to the future iterations of this experience and appreciation everyone taking the time to learn on the job. 
 
![Screenshot 2024-09-16 at 2 25 34 PM](https://github.com/user-attachments/assets/deb01cb7-9709-46c8-a0ec-3aa1955d4beb)

_Drink water in my GitHub Larq bottle._

**Comment by nickytonline** [comment]
Great work everyone! I can say a bunch of us are Gophers now! 😆

<img src="https://media2.giphy.com/media/Swn5yuXfDhg40/giphy.gif"/>

Big ups to @open-sauced/staff and big ups

- @jpmcb for leading this and providing us with great resources to get ramped up on Go.
- Great work on the interactive CLI @zeucapua! ::chefs_kiss::
- Great work on all the doc updates, content, and invaluable feedback for the CLI @BekahHW!
- @brandonroberts you are officially a Gopher bug squasher!
- @bdougie for the great feedback and story telling as always.
- @isabensusan for landing page designs and always thinking about the user

![Photo on 2024-09-16 at 6 02 PM #3](https://github.com/user-attachments/assets/e05240c0-3ea4-4f92-a7fb-5fc57a517693)

**Comment by BekahHW** [comment]
![image](https://github.com/user-attachments/assets/f649f379-43bd-4a3d-b719-0297856e35d3)


We put together something really awesome and useful in a matter of weeks. A huge shoutout to the whole team. 

@jpmcb for being a fantastic launch captain, making sure we were on track, being thoughtful about the approach and process, and scoping this out. Seeing this feature go from your POC to where it is now is a testament to your ability to lead and support the team. 

@nickytonline great work becoming a gopher! Also, your GH action is a great addition to make this a valuable tool. 

@zeucapua the interactive CLI decreased the friction to get started and improved the user experience by so much. Amazing work. 

@brandonroberts for making sure we had all our bases covered and playing utility. 

@isabensusan for making sure I had the assets to be able to share the story with others. 

@bdougie for trusting the team with a new challenge that can make an impact. 

Cheers 🥂

**Comment by brandonroberts** [comment]
We came, we gophered, we launched 🫡 

@jpmcb for cranking on the Pizza CLI, steering the ship as launch captain, and introducing us to more Go!

@nickytonline for jumping to Gopher-land with me again!

@zeucapua for shipping the interactive CLI work!

@isabensusan for designs, feedback, and thumbnails!

@BekahHW for all the content and coordination across our channels!

@bdougie for enabling all of us to make impact!

@chhristopher for producing quality content and making things work in the background!

![Photo on 9-17-24 at 10 26 AM](https://github.com/user-attachments/assets/9932973f-bfe6-466e-bdf7-b85d01d2b778)

🍕
